### PR TITLE
n/a

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php [hunk 1]
```diff
@@ -1,4 +1,7 @@
-<?php
+       
+       
+       
+       <?php
```
Comment (stof) [inline src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php]: why this change ?

File: src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php [hunk 2]
```diff
@@ -51,10 +54,11 @@ public function testProcessFailedExceptionPopulatesInformationFromProcessOutput(
         $exitText = 'General error';
         $output = 'Command output';
         $errorOutput = 'FATAL: Unexpected error';
+		$workingDirectory = getcwd ();
```
Comment (drmonty) [inline src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php]: tab -> space

File: src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php [hunk 3]
```diff
@@ -81,27 +85,32 @@ public function testProcessFailedExceptionPopulatesInformationFromProcessOutput(
...
      * Tests that ProcessFailedException does not extract information from
-     * process output if it was previously disabled
+     * process output if it was previously disabled.
      */
     public function testDisabledOutputInFailedExceptionDoesNotPopulateOutput()
     {
         $cmd = 'php';
         $exitCode = 1;
         $exitText = 'General error';
+		$workingDirectory = getcwd ();
```
Comment (drmonty) [inline src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php]: same here

File: src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php [hunk 4]
```diff
@@ -126,10 +135,14 @@ public function testDisabledOutputInFailedExceptionDoesNotPopulateOutput()
             ->method('isOutputDisabled')
             ->will($this->returnValue(true));
 
+		$process->expects($this->once())
```
Comment (drmonty) [inline src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php]: and here

**Comment by fabpot** [comment]
That was a test to debug fabbot.io.
